webapp/src/app/shared/phaser/map-pan.ts

```diff
@@ -26,7 +28,7 @@ export class MapPan extends Phaser.GameObjects.GameObject {
 	}
 	
 	private onMouseDown() {
-		if (!this.active) {
+		if (!this.active || !Globals.panning){
```

Comment:
I'm pretty sure this line prevents middle mouse panning to work

Reply:
@Vegita2 I'll take a look, and find my mouse to test with that as well

Reply:
@Vegita2 Fixed

Reply:
Disregard,  this fixed the issue with the mouse wheel,  but created a different issue.  Will ping you again when this is ready

Reply:
@Vegita2 Okay,  fixed fixed